src/components/Page/IaC/IaC.test.tsx

```diff
@@ -26,46 +26,6 @@ describe('IaC component', () => {
 		}
 	}
 
-	test('renders IaC component with title and severity', () => {
-		const { getByText } = render(<IaC data={mockData} />)
-
-		expect(getByText('IaC Vulnerability')).toBeInTheDocument()
-		expect(getByTextAcrossMultipleElements(getByText, 'Severity: High'))
-	})
-
-	test('renders IaC component with location and abbreviation', () => {
-		const { getByText } = render(<IaC data={mockData} />)
-
-		expect(getByTextAcrossMultipleElements(getByText, 'Location: example-file.tf: 10'))
-		expect(getByTextAcrossMultipleElements(getByText, 'Abbreviation: ABC'))
-	})
-
-	test('renders IaC component with description and code snippet', () => {
-		const { getByText } = render(<IaC data={mockData} />)
-
-		expect(getByText('Description')).toBeInTheDocument()
-		expect(getByText('This is a vulnerability description.')).toBeInTheDocument()
-		expect(getByText('Example code snippet')).toBeInTheDocument()
-	})
-
-	test('renders IaC component with finding, guidance, and meaning', () => {
-		const { getByText } = render(<IaC data={mockData} />)
-
-		expect(getByText('FINDING')).toBeInTheDocument()
-		expect(getByText('What should I do?')).toBeInTheDocument()
-		expect(getByText('What can happen?')).toBeInTheDocument()
-		expect(getByText('What Does it mean?')).toBeInTheDocument()
-	})
-
-	test('renders IaC component with code snippet, how it happens, and what to do', () => {
-		const { getByText } = render(<IaC data={mockData} />)
-
-		expect(getByText('Example code snippet')).toBeInTheDocument()
-		expect(getByText('How it happens')).toBeInTheDocument()
-		expect(getByText('Meaning of the vulnerability')).toBeInTheDocument()
-		expect(getByText('What to do about it')).toBeInTheDocument()
-	})
-
```

Comment:
Let's add new tests in place of those, which cover the visibility of InformationTabs and Header